tools/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Dockerfile for the KDC image
```

Comment:
out of scope: I'm pretty sure this should be removed from the upstream ...

Reply:
Agreed, just a blind copy here